chapter15/url2png/package.json

```diff
@@ -1,11 +1,11 @@
 {
  "dependencies": {
-  "aws-sdk": "2.369.0",
-  "uuid": "3.3.2",
-  "node-webshot": "1.0.3"
```

Comment:
Phantom.js is no longer maintained and was not working on Amazon Linux 2 and Node.js 14. The results of puppeteer + chromium are impressive!